var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -173,3 +187,15 @@ def check(self):
         since they are already parallelized"""
         with working_dir(self.build_directory):
             self._if_ninja_target_execute("test", parallel=False)
+
+    @run_after("install")
+    def install_modfiles(self):
+        if self.spec.satisfies("%fj"):
+            for mod in glob.iglob(self.build_directory+"/src/*.mod"):
+                install(mod, self.prefix.include)
+
+    def flag_handler(self, name, flags):
+        if self.spec.satisfies("%fj"):
+            if name == "fflags":
+                flags.append("-Kparallel -Kopenmp -Free")
+        return (None, None, flags)
```

Comment:
That's the part which should go into https://github.com/cp2k/dbcsr/blob/develop/cmake/CompilerConfiguration.cmake (see comment above).